tendrl/node_agent/discovery_modules/plugin.py

```diff
@@ -0,0 +1,11 @@
+class PluginMount(type):
```

Comment:
I added in commons keeping in mind that in future if sds integration modules also introduce some plugins mechanism, this meta class could be used.

Also meta class doesnt restrict uses of only plugin but the way we get and execute the plugin. All that meta class provides is that we can load and maintain a list of plugins. Later stage using the interface name for the plugin we loop through the plugins and decide what to do.

Still I leave to team, may be we can have at discovery module level only.